src/languages/handlebars.js

```diff
@@ -6,63 +6,91 @@ Description: Matcher for Handlebars as well as EmberJS additions.
 Website: https://handlebarsjs.com
 Category: template
 */
-function(hljs) {
+function (hljs) {
```

Comment:
This kind of change just bit me today. :)  Although a few of the other languages are also so different.